### ISSUE TITLE
Support for limit and offset in list_in_collection

### ISSUE THREAD

**Issue by mwvaughn** [TACC/paas#13]
According to the OpenAPI spec (and based on functional tests), it seems like *limit* and *offset* support are missing from `list_in_collection`.  

```shell
tapis3 pg tables list organization
+---------------------------------------+-------+-----------------+----------------------+-------+
| description                           | name  | organization_id | url                  | _pkid |
+---------------------------------------+-------+-----------------+----------------------+-------+
| Data Coordinating Center              | TACC  |               1 | tacc.utexas.edu      |     1 |
| University of Illinois at Chicago     | UIC   |               2 | uic.edu              |     2 |
| Rush University Medical Center        | Rush  |               3 | rush.edu             |     3 |
| Institute for Translational Medicine  | ITM   |               4 | chicagoitm.org       |     4 |
| University of Chicago Medical Center  | UCMC  |               5 | uchicagomedicine.org |     5 |
| NorthShore                            | NS    |               6 | northshore.org       |     6 |
| University of Michigan                | UMICH |               7 | umich.edu            |     7 |
| University of Iowa                    | UIOWA |               8 | uiowa.edu            |     8 |
| Pacific Northwest National Laboratory | PNNL  |               9 | pnnl.gov             |     9 |
| University of California San Diego    | UCSD  |              10 | ucsd.edu             |    10 |
+---------------------------------------+-------+-----------------+----------------------+-------+
```
There are > 10 records in this collection. Sending a `limit` kwarg to `list_in_collection` does not change the response.

**Comment by NotChristianGarcia** [comment]
Fixed in #15.

**Comment by NotChristianGarcia** [comment]
Fixed in #15.
